### PR TITLE
Axios

### PR DESCRIPTION
**Pull request by amik888**

### CODE REVIEW

File: backend/package.json [hunk 1]
```diff
@@ -18,11 +18,11 @@
     "express": "^4.17.1",
     "forever": "^3.0.4",
     "mysql2": "^2.2.5",
+    "react-scripts": "^4.0.3",
     "sequelize": "^6.6.2"
   },
   "devDependencies": {
-    "jest": "^26.6.3",
```
Comment (ahrayahlabcoimin) [inline backend/package.json:24]: Why is jest removed?

**Review by ahrayahlabcoimin — CHANGES_REQUESTED** [review]
You removed some of the testing modules. The changes made would also break the tests that I created.

File: backend/package.json (hunk 1 above)
Comment (amik888) [reply backend/package.json:24]: I was having trouble running npm start and the error response suggested I remove it. When I removed it, I was able to run npm start

File: hello-world-example/src/server.js [hunk 2]
```diff
@@ -3,11 +3,15 @@ const cors = require('cors;');
 const app = express();
 
 app.use(cors());
-
-app.use('login', (req, res) => {
-    res.send({
-        token: 'test123'
-    });
+// simple route
```
Comment (ahrayahlabcoimin) [inline hello-world-example/src/server.js:6]: I do not know why this is in frontend. I know you did not introduce this but I think this entire file should be in the backend.

**Review by ahrayahlabcoimin — CHANGES_REQUESTED** [review]

File: backend/package.json (hunk 1 above)
Comment (ahrayahlabcoimin) [reply backend/package.json:24]: No idea why it would fix that, but that is necessary to run tests.

**Review by dragon9hide — APPROVED** [review]

**Pull request merged**